scripts/rest_examples/reservation-create-gcp.http

```diff
@@ -10,7 +10,6 @@ X-Rh-Identity: {{identity}}
   "image_id": "https://www.googleapis.com/compute/v1/projects/rhel-cloud/global/images/rhel-9-v20230411",
   "amount": 1,
   "machine_type": "e2-micro",
-  "launch_template_id": "{{launch_template_id}}",
```

Comment:
Intentional? Just checking, might be useful.

Reply:
Just removed for testing, commited by mistake, thanks for catching that! :orange_heart: